### ISSUE TITLE
Access another drive (Windows)

### ISSUE THREAD

**Issue by bastiqui** [vKolerts/sidenoder#15]
Hello:

I wanted to share that I'm not able to change the mount point, either access another drive like "H:".
I've tried to create a symlink but whenever it creates the "mnt" folder in "%AppData%/Temp" it cannot duplicate the symlink so I just can't access to the folder I want to go.

Here's the folder created, I've used "mklink" with all the three options "/D, /J, /H".
![image](https://user-images.githubusercontent.com/12750310/146769372-70cc958f-d9a3-4a7c-a417-dc56ec845d55.png)

And here's what the "mnt" has created.

![image](https://user-images.githubusercontent.com/12750310/146768995-18e0c2fa-9898-4cc9-8583-88fb1da720d8.png)

And of course, from the program I'm not able or I don't know how to change the drive:

![image](https://user-images.githubusercontent.com/12750310/146769558-fd65275a-f1b9-4094-948f-044653388412.png)

**Comment by KaladinDMP** [comment]
It seems like to me you are trying to use the program for something besides it's intended use. Rclone, rclone browser, or something similar seems more like what you are looking for. Are these drives network drives or local drives? Network drives aren't likely to show up in this situation without making them a remote through the rclone config... Which is what this program uses as well. So my point being: learn to use rclone config will teach you more if this program will be able to do what you want it to or if you may need to find a new solution.

**Comment by vKolerts** [comment]
Sorry but why you want to change mount point? I can make mount path customizable, but I don't understand what purpose required that.
About accessing to other discs - I check it later on Windows. I think, you can add manualy other discs to bookmarks by {user_folder}/sidenoder/config.json.  Thanks for reminding

**Comment by bastiqui** [comment]
Hello:

No KaladinDMP, that's not what I want to say, I want to be able to the change the mount point and accessing other disks as vKolerts said.

And vKolerts, I want to change the mount point because I want to leave C:\ clean, I use an external HDD where I like to put my things on.

And thanks for the help, now I've added a new "dirBookmarks":
![image](https://user-images.githubusercontent.com/12750310/146814963-217b680c-de28-41f9-bfbd-ab2948219b8d.png)


From the program I'm able to go to the bookmark I want to:
![image](https://user-images.githubusercontent.com/12750310/146815038-ec3925bc-837c-4373-b4d3-13a6682f838f.png)

And it works.
![image](https://user-images.githubusercontent.com/12750310/146815458-3c2e5f3f-d853-4532-b196-dfd5e1eb42df.png)

Amazing software, this is exactly what I wanted something offline for the resilio folder backup I've got.
